core/src/consensus.rs

```diff
@@ -60,27 +60,12 @@ impl SwitchForkDecision {
 pub const VOTE_THRESHOLD_DEPTH: usize = 8;
 pub const SWITCH_FORK_THRESHOLD: f64 = 0.38;
 
-#[derive(Default, Debug, Clone)]
-pub struct StakeLockout {
-    lockout: u64,
-    stake: u64,
-}
-
-impl StakeLockout {
-    pub fn new(lockout: u64, stake: u64) -> Self {
-        Self { lockout, stake }
-    }
-    pub fn lockout(&self) -> u64 {
-        self.lockout
-    }
-    pub fn stake(&self) -> u64 {
-        self.stake
-    }
-}
+pub type Stake = u64;
+pub type VotedStakes = HashMap<Slot, Stake>;
 
 pub(crate) struct ComputedBankState {
-    pub stake_lockouts: HashMap<Slot, StakeLockout>,
-    pub total_staked: u64,
+    pub voted_stakes: VotedStakes,
+    pub total_stake: Stake,
```

Comment:
@carllin I've ended up renaming bunch of interrelated fields/types while addressing https://github.com/solana-labs/solana/pull/10719/files#r443351567. How does this look for you, overall?

Also, I've chosen `adjuctive noun` naming convension because I think this is more natural than `total_staked`/`vote_staked`, etc. And the precedent exists: `rooted_stake` in the existing code (not `stake_rooted`) and `Active Stake: XXX` in `solana validator` (not `Active Staked`).

Reply:
Ah, also, I've added `Voted` intentionally as just `Stakes` sounded too general.

Reply:
@ryoqun fine by me!